__tests__/proxy.test.ts

```diff
@@ -39,72 +39,112 @@ describe('proxy', () => {
         })
     })
 
-    it('does not return proxyUrl if variables not set', () => {
+    it('getProxyUrl does not return proxyUrl if variables not set', () => {
```

Comment:
To simplify the unit tests related to bypass scenarios, I added another exported function `checkBypass(string)` to proxy.ts

Therefore I prefixed the test names to clarify which function was being exercised.